### PR TITLE
Added Hash Function

### PR DESCRIPTION
**Pull request by sachinam1397**
Added Hash Function, which generates a hash of length 64.

### CODE REVIEW

File: Server/Hash/hash_function.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+			ord_encryption += ord(password[i])
+			final_hash += str(ord(password[i])%len(password))
+
+		value_encrypting = 0
+		get_encrypted_value_string = ""
+		for i in range(1,len(password)):
+			value_encrypting += max(ord(password[i]),ord(password[i-1])) - min(ord(password[i]),ord(password[i-1]))
+			temp = max(ord(password[i]),ord(password[i-1])) - min(ord(password[i]),ord(password[i-1]))
+			value_temp = max(ord(password[i]),ord(password[i-1])) - min(ord(password[i]),ord(password[i-1]))
+			if(value_temp > 32 and value_temp < 128):
```
Comment (Valiant1011) [inline Server/Hash/hash_function.py:19]: what is the purpose of this line? does it ignore other characters?

Comment (sachinam1397) [reply Server/Hash/hash_function.py:19]: this is because from ASCII value 1 to 32 character is blank so to ignore blank characters this line is used

Comment (Valiant1011) [reply Server/Hash/hash_function.py:19]: someone may have a space in their password, that is ASCII 32

Comment (sachinam1397) [reply Server/Hash/hash_function.py:19]: That will not have any effect, this is only to avoid space in generated hash.

**Pull request merged**